raiden-dapp/src/views/TransferRoute.vue

```diff
@@ -40,15 +43,47 @@ import { Token, TokenModel } from '@/model/types';
     NoChannelsDialog,
   },
   computed: {
+    ...mapState(['latestStateBackupReminder']),
     ...mapGetters(['tokens', 'channelWithBiggestCapacity', 'openChannels']),
   },
 })
 export default class TransferRoute extends Vue {
+  latestStateBackupReminder!: number;
   tokens!: TokenModel[];
+  currentTime: number = new Date().getTime();
+  oneDay: number = new Date().setHours(24);
+  stateBackupReminder = {
```

Comment:
The same as before. Either you say this is something fixed and do it outside, or it is always new for each notification and create in inside of the function. But it will never change. So it is not a state.

PS: Once we have resolved the messy RaidenService module we should have proper notification helper that reduce the code here and keep if more focused (in fact this component should not care about how the notification looks in detail, just send it).